clippy_dev/src/lib.rs

```diff
@@ -89,9 +92,9 @@ fn parse_contents(content: &str, filename: &str) -> impl Iterator<Item=Lint> {
 }
 
 /// Collects all .rs files in the `clippy_lints/src` directory
-fn lint_files() -> impl Iterator<Item=fs::DirEntry> {
-    fs::read_dir("../clippy_lints/src")
-        .unwrap()
+fn lint_files() -> impl Iterator<Item=walkdir::DirEntry> {
+    WalkDir::new("../clippy_lints/src")
```

Comment:
can you add a comment here that we use `WalkDir` instead of `fs::read_dir` to recurse into the directories?

Reply:
yup!